packages/search-ui-elasticsearch-connector/package.json

```diff
@@ -41,6 +41,6 @@
   "dependencies": {
     "@elastic/elasticsearch": "^8.2.1",
     "@elastic/search-ui": "1.15.0",
-    "@searchkit/sdk": "^3.0.0-canary.57"
+    "@searchkit/sdk": "^3.0.0-canary.58"
```

Comment:
Why do you use the canary versions in SearchKit?

Reply:
searchkit v3 not yet pass pre-release yet so still on canaries right now.